travis-ci.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+# Don't fail silently when a step doesn't succeed
+set -e
+
+make container deploy_kind
+
+echo "|---- Creating new Sonobuoy run/waiting for results..."
+./sonobuoy run --image-pull-policy=IfNotPresent -m Quick --wait
+
+outFile=$(./sonobuoy retrieve)
+results=$(./sonobuoy e2e $outFile)
+
+echo $results
+
+if echo $results | grep --quiet "failed tests: 0"; then
```

Comment:
This is not enough condition to declare the test OK.
I have the same result in a failed deployment 
````
{"level":"info","msg":"Collecting Pod Logs...","time":"2019-03-05T12:35:40Z"}
{"level":"error","msg":"the server rejected our request for an unknown reason (get pods sonobuoy-e2e-job-27d5ca4105034c28)","
time":"2019-03-05T12:35:40Z"}
```

and failed tests :0 but the problem that it didn't run any test
```
sonobuoy e2e 201903051232_sonobuoy_7cd48d20-e974-42dd-8a4b-1dbcf0ae3bd2.tar.gz
failed tests: 0
```